### DISCUSSION TITLE
Clarification regarding Request-Transformer-Plugin & Response-Transformer-Plugin

### DISCUSSION THREAD

**Discussion by Zili-H** [Kong/kong#13685] [Help] [answered]
Hi, 

I have a couple of questions regarding both plugins since some usecases are unclear to me and not fully documented:

Request-Transformer: https://docs.konghq.com/hub/kong-inc/request-transformer/
- http_method
  - ~~If left empty, the transformation applies to all methods, right?~~ 
 >| this field changes the method to the specified value.
  - ~~This field only supports a single http_method or is there a way to specify muliple?~~
  -  In the [example documentation](https://docs.konghq.com/hub/kong-inc/request-transformer/how-to/basic-example/), we can see this annotation ` "qs2-old-name:qs2-new-name"` in the remove action.
      For remove action, could you please tell me why in the example here a field value is specified, the value after the double colon :  
     From my tests, the fieldvalue for headers, body and querystring are ignored and not necessary and only a field name/path should be specified for the remove action.

Response-Transformer: 
- ~~Json_types~~
  - ~~this field can be found in e.g. replace action~~
   ~~If i add 2 entries to the replace.json field with this as value: ` ["open:true", "position:1"]`, how do I specify that the first value is of jsonType boolean and the second number? What would the value for json_types be?~~
  - ~~I glimpsed into the source code, apparently json_types will try to cast the value in this order, number, boolean and finally string~~  
>| Each entry matches with each individual json value

Following a thorough testing, I'm able to clarify most questions except the one that is not struckthrough

Best regards

**Comment by StarlightIbuki** [comment, marked as answer]
> In the [example documentation](https://docs.konghq.com/hub/kong-inc/request-transformer/how-to/basic-example/), we can see this annotation  "qs2-old-name:qs2-new-name" 

I believe that is a misdoumentation. Looks like it's a copy&paste. PRs are welcome.

**Reply by Zili-H** [reply]
Alright, thank you for the answer.
